cinema/urls.py

```diff
@@ -1,10 +1,26 @@
-from django.urls import path
+from django.urls import path, include
+from rest_framework import routers
 
-from cinema.views import movie_list, movie_detail
+from cinema.views import (
+    GenreList,
+    GenreDetail,
+    ActorList,
+    ActorDetail,
+    CinemaHallViewSet,
+    MovieViewSet
+)
+
+routers = routers.DefaultRouter()
+
+routers.register("cinema_halls", CinemaHallViewSet)
+routers.register("movies", MovieViewSet)
 
 urlpatterns = [
-    path("movies/", movie_list, name="movie-list"),
-    path("movies/<int:pk>/", movie_detail, name="movie-detail"),
+    path("", include(routers.urls)),
```

Comment:
No need to include router when basepath is empty. Use:
urlpatterns = [list of urls] + router.urls